src/Tribe/Tickets.php

```diff
@@ -621,15 +621,48 @@ public function save_ticket( $post_id, $ticket, $raw_data = [] ) {
 		}
 
 		/**
-		 * Returns all the tickets for an event.
+		 * Returns all the tickets for an event, of the active ticket providers.
 		 *
-		 * @abstract
+		 * @since TBD Changed from protected abstract to public with duplicated child classes' logic consolidated here.
 		 *
 		 * @param int $post_id ID of parent "event" post.
 		 *
 		 * @return Tribe__Tickets__Ticket_Object[] List of ticket objects.
 		 */
-		protected function get_tickets( $post_id ) {}
+		public function get_tickets( $post_id ) {
+			$default_provider = Tribe__Tickets__Tickets::get_event_ticket_provider( $post_id );
```

Comment:
:no_entry_sign: Must use "self::" for local static member reference

<sub>hash: 2e7ebe326ceb3a32cd407ea004627df4</sub>